src/ansys_sphinx_theme/__init__.py

```diff
@@ -1,55 +1,116 @@
-pyansys_logo_white = os.path.join(_this_path, "static", "pyansys-logo-white-cropped.png")
-ansys_favicon = os.path.join(_this_path, "static", "ansys-favicon.png")
-ansys_logo_white = os.path.join(_this_path, "static", "ansys_logo_white.pdf")
-ansys_logo_white_cropped = os.path.join(_this_path, "static", "ansys_logo_white_cropped.pdf")
-watermark = os.path.join(_this_path, "static", "watermark.pdf")
-ansys_logo_black = os.path.join(_this_path, "static", "ansys_logo_black_cropped.jpg")
+ansys_favicon = str((STATIC_PATH / "ansys-favicon.png").absolute())
+ansys_logo_black = str((STATIC_PATH / "ansys_logo_black_cropped.jpg").absolute())
+ansys_logo_white = str((STATIC_PATH / "ansys_logo_white.pdf").absolute())
+ansys_logo_white_cropped = str((STATIC_PATH / "ansys_logo_white_cropped.pdf").absolute())
+html_logo = ansys_logo_black
+page_404 = str((STATIC_PATH / "404.rst").absolute())
+pyansys_logo_black = str((STATIC_PATH / "pyansys-logo-black-cropped.png").absolute())
+pyansys_logo_white = str((STATIC_PATH / "pyansys-logo-white-cropped.png").absolute())
+watermark = str((STATIC_PATH / "watermark.pdf").absolute())
+
+
+def get_html_theme_path() -> pathlib.Path:
+    """Return list of HTML theme paths.
 
-# Enable default 404 page
-page_404 = os.path.join(_this_path, "static", "404.rst")
+    Returns
+    -------
+    pathlib.Path
+        Path pointing to the installation directory of the theme.
 
-html_logo = pyansys_logo_black
-CSS_FILENAME = "ansys_sphinx_theme.css"
+    """
+    return THEME_PATH.resolve()
 
 
-def get_html_theme_path():
-    """Return list of HTML theme paths."""
-    return Path(__file__).parents[0].absolute()
+def get_version_match(semver: str) -> str:
+    """Evaluate the version match for the multi-documentation.
 
+    Parameters
+    ----------
+    semver : str
+        Semantic version number in the form of a string.
 
-def get_version_match(semver):
-    """Evaluate the version match for the multi-documentation."""
+    Returns
+    -------
+    str
+        Matching version number in the form of a string.
+
+    """
     if semver.endswith("dev0"):
         return "dev"
     major, minor, _ = semver.split(".")
     return ".".join([major, minor])
 
 
-def setup(app):
-    """Connect to the sphinx theme app."""
+def setup_default_html_theme_options(app):
+    """Set up the default configuration for the HTML options.
+
+    Parameters
+    ----------
+    app : sphinx.application.Sphinx
+        Application instance for rendering the documentation.
+
+    Notes
+    -----
+    This function is the only way to overwrite ``pydata-sphinx-theme``
+    configuration. Variables declared in the ``theme.conf`` do not include
+    inherited ones.
+
+    """
+    # Place all switchers and icons at the end of the navigation bar
+    app.config.html_theme_options.setdefault(
+        "navbar_end", ["version-switcher", "theme-switcher", "navbar-icon-links"]
+    )
```

Comment:
This fuction is executed when building the docs. If no configuartion is found for the `navbar_end`, this one applies.